style.css

```diff
@@ -448,3 +489,501 @@ footer .contact-para {
   margin-top: 18px;
   font-size: 18px;
 }
+
+@media screen and (min-width: 1050px) {
+  .first-page {
+    border-bottom-left-radius: 150px;
```

Comment:
@elmar8287 
Hello sir, what are you talking about? I use a laptop. but there is no problem with the footer. Please check out the screenshot of my full desktop screen.

Waiting for your reply


![2022-02-18_17-02-44](https://user-images.githubusercontent.com/78845635/154679812-dd32e975-ba19-4547-b738-c210b0de47fe.jpg)
 any sort of problem in the footer.

Reply:
@elmar8287,
Can you please reply over my question.

Reply:
@elmar8287 hello sir

Reply:
Dear @safar1212, I am telling about, as I mentioned an optional suggestion regarding the contact form is the displaying of inputs: you can see on my screenshot, that the width of inputs is smaller than textarea:
![2022-02-18 17 15 36](https://user-images.githubusercontent.com/49064106/154689690-0a0fa0de-8267-4acd-9484-11a7f26271ec.gif)

I am open for any support you need 👍

Reply:
@elmar8287 
I understand. But I want to say that, on my laptop, this seems okay. This might be due to the larger screen, as you might be using a desktop. So, how to solve that?

waiting for your reply

Reply:
@safar1212 maybe you can go on with using flexbox, instead of a grid in this contact form. 👍